csrankings-y.csv

```diff
@@ -8,6 +8,7 @@ Y. Raghu Reddy,IIIT Hyderabad,http://faculty.iiit.ac.in/~raghu.reddy,NOSCHOLARPA
 Y. Tsiatouhas,University of Ioannina,https://cs.uoi.gr/~tsiatouhas,gHsE8pYAAAAJ
 Y. Y. Yao,University of Regina,http://www.cs.uregina.ca/People/Profiles/YiyuYao.html,_aL0fcQAAAAJ
 Y.N. Srikant,IISc Bangalore,http://www.csa.iisc.ac.in/~srikant,NOSCHOLARPAGE
+Ya Zhang 0002,Shanghai Jiao Tong University,https://mediabrain.sjtu.edu.cn/yazhang,pbjw9sMAAAAJ

```

Comment:
Not in CS. Please clarify.

Reply:
Prof. Ya Zhang is affiliated with Cooperative Medianet Innovation Center(https://cmic.sjtu.edu.cn/EN/Default.aspx) of Shanghai Jiao Tong University, which is an interdisciplinary research center. She supervises students in EE as well as CS. But unfortunately, I could not supply any homepage showing a courtesy appointment in CS for her. In Shanghai Jiao Tong University, we do not need to hold a courtesy appointment in CS in order to supervise student in CS. Instead, we only need to apply for being a Ph.D supervisor for CS as the second discipline.  The page (https://faculty.sjtu.edu.cn/zhangya/zh_CN/zsxx/184191/content/54663.htm) shows that she can supervise Ph.D students in CS. Her work is also more related to CS as can be seen from her publication venue (CVPR, ICCV, ECCV, AAAI etc.)